snakePipes/shared/defaults.yaml

```diff
@@ -7,7 +7,7 @@
 # permitted here.
 ################################################################################
 snakemakeOptions: ' --use-conda --conda-prefix /package/anaconda3/envs/ '
-tempDir: /data/extended/
+tempdir: /data/extended/
```

Comment:
Is there a reason you're changing this?

Reply:
Looks like a mistake in resolving merge conflicts. I'll fix that, thanks!